lib/node_modules/@stdlib/blas/base/sdsdot/examples/index.js

```diff
@@ -18,23 +18,15 @@
 
 'use strict';
 
-var randu = require( '@stdlib/random/base/randu' );
-var round = require( '@stdlib/math/base/special/round' );
-var Float32Array = require( '@stdlib/array/float32' );
-var sdsdot = require( './../lib' ).ndarray;
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' ).factory;
+var filledarrayBy = require( '@stdlib/array/filled-by' );
+var sdsdot = require( '@stdlib/blas/base/sdsdot' );
```

Comment:
```suggestion
var sdsdot = require( './../lib' );
```

We always use a relative path in the `examples/index.js` file.